### ISSUE TITLE
Accessibility enhancements

### ISSUE THREAD

**Issue by himedlooff** [cfpb/cf-pagination#4]
- [x] Voiceover does not speak the second form label that says "of x". It would be nice if we could get it to do that. (New idea: hide the second label with `aria-hidden="true"` and update the first label to read "Page number out of X total pages").
- [ ] Make paging easier by allowing people to bypass any content that comes before the results. Possible solutions:
  - Add a **skip to results** link.
  - Auto focus on results when using a pagination button or action.
    (The first part of this has been added in #18. The second half is to update focus with JS like this: http://www.bignerdranch.com/blog/web-accessibility-skip-navigation-links/)
- [x] Change label color within the gray from area to `@darkgray` for better contrast. The current color is failing color contrast tests.

**Comment by himedlooff** [comment]
We've done some work in cfgov-refresh to address the first issue of screen readers not reading all appropriate content. Refer to https://github.com/cfpb/cfgov-refresh/blob/gh-pages/_layouts/posts-paginated.html when updating this pattern.
